jbpm-persistence/jbpm-persistence-api/src/main/java/org/jbpm/persistence/api/integration/model/TaskOperationView.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.persistence.api.integration.model;
+
+import org.jbpm.persistence.api.integration.InstanceView;
+import org.kie.api.task.TaskEvent;
+import org.kie.api.task.TaskLifeCycleEventListener.AssignmentType;
+import org.kie.internal.task.api.model.TaskEvent.TaskEventType;
+
+
+public class TaskOperationView implements InstanceView<TaskEvent> {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Minor comment: Is this serial OK?

Reply:
Since the class is new addition, using default one (1) is fine. In fact, being new, whatever number we use will be fine.